front/app/utils/errorUtils.ts

```diff
@@ -1,7 +1,7 @@
 import { CLErrorsJSON } from 'typings';
 import messages from './messages';
 
-export function isCLErrorJSON(error) {
+export function isCLErrorJSON(error: any): error is CLErrorsJSON {
```

Comment:
I think it's fine to make `error` here unknown as this is literally what this method is checking.

Reply:
Yes, I tried this but it introduced new errors, extending the scope of this. So now I've changed the function (tests pass) but is has a TS strict error. 

But then our error types may change/improve with data loading so I/we should not spend much time on this.